base/services/w32time/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_executable(W32Time W32Time.c ntpclient.c)
+set_module_type(W32Time win32cui UNICODE)
+add_importlibs(W32Time advapi32 msvcrt kernel32 ntdll ws2_32)
```

Comment:
msvcrt kernel32 ntdll go last, in that order. Add new dlls on the left instead of the right.

Reply:
I will fix this. I was not aware of how this should work. Thanks.